app-backend/database/src/main/scala/com/azavea/rf/database/tables/Exports.scala

```diff
@@ -183,25 +183,27 @@ object Exports extends TableQuery(tag => new Exports(tag)) with LazyLogging {
 
     eo.flatMap({ exportOptions =>
 
-        val outDef = OutputDefinition(
-          crs = exportOptions.getCrs,
-          rasterSize = exportOptions.rasterSize,
-          render = Some(exportOptions.render),
-          crop = exportOptions.crop,
-          stitch = exportOptions.stitch,
-          source = exportOptions.source,
-          dropboxCredential = user.dropboxCredential
-        )
+      val outDef = OutputDefinition(
+        crs = exportOptions.getCrs,
+        rasterSize = exportOptions.rasterSize,
+        render = Some(exportOptions.render),
```

Comment:
I save `.some` for when a value needs to be lifted in `Option` and not `Some` (the subtype).

compiles:
```scala
( 1.some |@| List(1).headOption).map(_ + _)  // Some(2)
```
Doesn't compile:
```scala
( Some(1) |@| List(1).headOption).map(_ + _)
```